test.js

```diff
@@ -90,3 +90,17 @@ test('different pascalCase option values', t => {
 		{pFooBar: true, pObj: {pTwo: false, pArr: [{pThreeFour: true}]}}
 	);
 });
+
+test('return orginal array if they are not objects', t => {
```

Comment:
You're not returning the original *array*, you're returning the original *array elements*